flocx_market/tests/system/test_offer.py

```diff
@@ -0,0 +1,89 @@
+from flocx_market.db.sqlalchemy.offer_api import OfferApi
```

Comment:
This doesn't seem to be related to implementing the contract api. Maybe open an issue for adding new tests and then add it in a new pull request?